src/main/java/spoon/reflect/code/CtThisAccess.java

```diff
@@ -19,20 +19,9 @@
 
 /**
  * This code element defines an access to this.
- * 
+ *
  * @param <T>
- *            Type of this
+ * 		Type of this
  */
-public interface CtThisAccess<T> extends CtExpression<T> {
-
-	/**
-	 * Tells if this access must be qualified by the class it refers to.
-	 */
-	boolean isQualified();
-
-	/**
-	 * Sets this access to be qualified by the class it refers to.
-	 */
-	void setQualified(boolean qualified);
-
+public interface CtThisAccess<T> extends CtTargetedExpression<T, CtExpression<?>> {
```

Comment:
Why? `CtSuperAccess` extends `CtTargetedAccess` to stay compatible with older version of Spoon but that isn't the case for `CtThisAccess`.